common/config/rush/pnpm-lock.yaml

```diff
@@ -3032,10 +3032,11 @@ packages:
       '@types/plist': 3.0.2
       onigasm: 2.2.5
       plist: 3.0.4
+      typescript: 4.4.4
     dev: false
     name: '@rush-temp/tmlanguage-generator'
     resolution:
-      integrity: sha512-WM1WhCLQw/RzuttQrS7QutOHooQHzcxlNAI8Jv5FUgaPmH7OGl7G3m/dH8KcWE4vSQRteEKlCNqeHU8w1z152w==
```

Comment:
nit: lock file needs to be gitignored?

Reply:
Lock file are here to ensure we all use the same dependencies tree and we don't have the "it works on my machine" issue for developmenent. (or at least way less).

Reply:
Yep, it's intended to be checked in and serves that purpose. In fact, if you don't check it in, you will fail PR validation on the "check for changed files step". It is configured to be collapsed by default in diffs:

https://github.com/microsoft/cadl/blob/88a51f1b6d6489adb1de1516a60d3e0ce977b4e8/.gitattributes#L4

![image](https://user-images.githubusercontent.com/75470/141503227-df68fed5-fddd-43f8-8d4d-0d6dcbae57ff.png)